apps/core/lib/mix/tasks/core_db_task.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Mix.Tasks.Core.Resetdb do
+  @moduledoc """
+  Custom mix task to reset db.
+  ```
+    iex> mix core.resetdb
+  ```
+  """
+  use Mix.Task
+
+  def run(_) do
+    System.cmd(
+      "mix",
+      ["ecto.drop"],
+      stderr_to_stdout: true,
+      into: IO.stream(:stdio, :line)
+    )
+
+    System.cmd(
+      "mix",
+      ["ecto.create"],
+      stderr_to_stdout: true,
+      into: IO.stream(:stdio, :line)
+    )
+
+    System.cmd(
+      "mix",
+      ["ecto.migrate"],
+      stderr_to_stdout: true,
+      into: IO.stream(:stdio, :line)
+    )
+  end
+end
+
+defmodule Mix.Tasks.Core.Seed do
+  @moduledoc """
+    Custom mix task to seed db.
+    ```
+      iex> mix core.seed
```

Comment:
This is not run in an IEx shell :grinning:

Reply:
sorry my bad will change this.